lib/controllers/flow/account_linking_flow_controller.dart

```diff
@@ -47,7 +49,22 @@ class AccountLinkingFlowController extends GetxController {
     _startListening(documentId);
   }
 
-  Future<void> confirmAccounts() async {}
+  Future<void> initiateConsentRequest(List<Account> accsToLink) async {
+    _setAwaitingUpdate(true);
+
+    final Consent updated = Consent(
```

Comment:
nit but would it be better to name this `updatedConsent` to make the variable name clearer